Source/JavaScriptCore/assembler/LinkBuffer.cpp

```diff
@@ -66,7 +66,7 @@ LinkBuffer::CodeRef<LinkBufferPtrTag> LinkBuffer::finalizeCodeWithoutDisassembly
     ASSERT(m_didAllocate);
     CodeRef<LinkBufferPtrTag> codeRef(m_executableMemory ? CodeRef<LinkBufferPtrTag>(*m_executableMemory) : CodeRef<LinkBufferPtrTag>::createSelfManagedCodeRef(m_code));
 
-    if (UNLIKELY(Options::logJITCodeForPerf()))
+    if (UNLIKELY(Options::useJITDump()))
         logJITCodeForPerf(codeRef, simpleName);
```

Comment:
We need to rename `logJITCodeForPerf` function too?

Reply:
Changed it to logJITCodeForJITDump.